### PR TITLE
Migrate datapunt asc to amsterdam asc MERGE FIRST

### PR DESCRIPTION
**Pull request by jasperswart**
- removed @datapunt/asc
- added @amsterdam/asc

### CODE REVIEW

File: src/styles/SharedStyles.js [hunk 1]
```diff
@@ -12,3 +12,11 @@ export const IntroTextStyle = styled.p`
   font-size: 19px;
   line-height: 26px;
 `;
+
+export const AscH1 = styled(Heading)`
+  margin: ${themeSpacing(5)} 0;
+`;
+
+export const AscHeaderSecondary = styled(Heading)`
```
Comment (aditudorache) [inline src/styles/SharedStyles.js:20]: Same here, why not  `import {Heading as AscHeading}`  and use the own heading in the application?

File: src/views/concepts/ConceptPage.jsx [hunk 2]
```diff
@@ -2,22 +2,22 @@ import React from 'react';
...
     <ConceptPageStyle>
       <ContentStyle>
         <BlueLinkButton
-          to={appRoutes.HOME}
+          href={appRoutes.HOME}
           text="Terug naar kaart"
           chevronDirection="left"
         />
-        <h1>Begrippenlijst</h1>
+        <AscH1>Begrippenlijst</AscH1>
```
Comment (aditudorache) [inline src/views/concepts/ConceptPage.jsx:20]: Don't use Asc prefix

**Review by aditudorache — CHANGES_REQUESTED** [review]

**Review by aditudorache — APPROVED** [review]

File: src/styles/SharedStyles.js [hunk 3]
```diff
@@ -12,3 +12,11 @@ export const IntroTextStyle = styled.p`
   font-size: 19px;
   line-height: 26px;
 `;
+
+export const AscH1 = styled(Heading)`
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/styles/SharedStyles.js]: gedaan

File: src/views/concepts/ConceptPage.jsx (hunk 2 above)
Comment (jasperswart) [reply src/views/concepts/ConceptPage.jsx]: gedaan

File: src/styles/SharedStyles.js (hunk 1 above)
Comment (jasperswart) [reply src/styles/SharedStyles.js]: gedaan

**Pull request merged**